### PR TITLE
Add commands for pausing and resuming connections

### PR DESCRIPTION
**Pull request by gulopine**

### CODE REVIEW

File: lib/commands/connect/pause.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    ],
+    needsApp: true,
+    needsAuth: true,
+    run: cli.command(function (context, heroku) {
+      api.withUserConnections(context.auth.password, context.app, context.flags, false, heroku).then(function(connections){
+          if (connections.length > 1) {
+            console.log("Multiple connections found. Please use '-r' to specify a single connection by resource name.");
+            api.connection_info(connections);
+          } else {
+            connections.forEach(function(connection) {
```
Comment (sibson) [inline lib/commands/connect/pause.js]: forEach seems weird here but I presume thats an artifact of the API we don't control.

**Comment by sibson** [comment]
+1

File: lib/commands/connect/pause.js (hunk 1 above)
Comment (gulopine) [inline lib/commands/connect/pause.js]: Yeah, I wondered about that. I stole it from existing commands, basically. I don't like it, but I'd rather fix them all together than worry about it just for adding these new commands.

Comment (sibson) [inline lib/commands/connect/pause.js]: agree 100%

**Pull request merged**